test/3x/torch/quantization/test_static_quant.py

```diff
@@ -76,7 +76,7 @@ def test_static_quant_fallback(self):
         quant_config = get_default_static_config()
         example_inputs = self.input
         # fallback by op_type
-        quant_config.set_local(torch.nn.Linear, StaticQuantConfig(w_dtype="fp32", act_dtype="fp32"))
+        quant_config.set_local([torch.nn.Linear, "Linear&add"], StaticQuantConfig(w_dtype="fp32", act_dtype="fp32"))

```

Comment:
Please add some UTs to ensure that all `Linear&add` are fallbacked as expected.

Reply:
It's added already. The message is dumped in log and verified manually.